types/office-js-preview/index.d.ts

```diff
@@ -20928,7 +20928,7 @@ declare namespace Office {
         cancelLabel?: string;
         /**
          * When you use the {@link https://learn.microsoft.com/javascript/api/outlook/office.mailboxevent#outlook-office-mailboxevent-completed-member(1) | completed method} to signal completion of an event handler and set its `allowEvent` property to `false`,
-         * this property specifies the ID of the task pane that opens when the **Don't Send** button is selected from the Smart Alerts dialog.
+         * this property specifies the ID of the task pane or function that opens when the **Don't Send** button is selected from the Smart Alerts dialog.
```

Comment:
I noticed that you changed this in office-js but didn't change it in office-js-preview. Let's keep them in sync

```suggestion
         * this property specifies the ID of the task pane or function that runs when the **Don't Send** button is selected from the Smart Alerts dialog.
```